models/labels/addresses/__single_category_labels__/balancer_v1/labels_balancer_v1_pools_ethereum.sql

```diff
@@ -13,46 +14,55 @@
 
 WITH events AS (
     -- binds
-    SELECT call_block_number AS block_number, 
-    contract_address AS pool, token, denorm
+    SELECT call_block_number AS block_number,
+           contract_address  AS pool,
+           token,
+           denorm
     FROM {{ source('balancer_v1_ethereum', 'BPool_call_bind') }}
     WHERE call_success
     {% if is_incremental() %}
-        AND call_block_time >= date_trunc("day", now() - interval '1 week')
+        AND call_block_time >= date_trunc('day', now() - interval '7' day)
     {% endif %}
 
     UNION all
 
     -- rebinds
-    SELECT call_block_number AS block_number, 
-    contract_address AS pool, token, denorm
+    SELECT call_block_number AS block_number,
+           contract_address  AS pool,
+           token,
+           denorm
     FROM {{ source('balancer_v1_ethereum', 'BPool_call_rebind') }}
     WHERE call_success
     {% if is_incremental() %}
-        AND call_block_time >= date_trunc("day", now() - interval '1 week')
+        AND call_block_time >= date_trunc('day', now() - interval '7' day)
     {% endif %}
 
     UNION all
-    
+
     -- unbinds
-    SELECT call_block_number AS block_number, 
-    contract_address AS pool, token, 0 AS denorm
+    SELECT call_block_number AS block_number,
+            contract_address AS pool,
+            token,
+            uint256 '0' AS denorm
     FROM {{ source('balancer_v1_ethereum', 'BPool_call_unbind') }}
     WHERE call_success
     {% if is_incremental() %}
-        AND call_block_time >= date_trunc("day", now() - interval '1 week')
+        AND call_block_time >= date_trunc('day', now() - interval '7' day)
     {% endif %}
 ),
 
 state_with_gaps AS (
-    SELECT events.block_number, events.pool, events.token, CAST(events.denorm AS double),
+    SELECT events.block_number
+           , events.pool
+           , events.token
+           , CAST(events.denorm AS double) AS denorm,
```

Comment:
on line 46, you cast to `uint256`, but do `double` here -- do we know what we want the output to be?

Reply:
I am not sure with this field actually.
I have casted the denorm into `uint256` type.

`denorm` is `uint256` type in other decoded tables:
https://dune.com/queries?blockchain=ethereum&category=decoded_project&namespace=balancer_v1&contract=BPool&blockchains=ethereum&abi=rebind

Reply:
nice, then this is the right call